src/app/common/http/common.http.types.ts

```diff
@@ -4,12 +4,15 @@ export type TDataState<T> = {
   data: T;
   isLoading: false;
   hasError: false;
+  nowEditWallet: number;
```

Comment:
This is generic type, should not be extended with properties specific for single component. Instead you should have created property `public nowEditWallet: number` inside my-wallets component.